zerver/openapi/zulip.yaml

```diff
@@ -19895,6 +19897,32 @@ paths:
                               type: integer
                               description: |
                                 The user group's integer ID.
+                            date_created:
+                              type: integer
+                              nullable: true
+                              description: |
+                                The UNIX timestamp for when the user group was created, in UTC seconds.
+
+                                A `null` value means the user group has no recorded date, which is often
+                                because the user group is very old, or because it was created via a data
```

Comment:
"often because the group predates the metadata being tracked starting in Zulip 8.0".-- unlike channels, we haven't had audit logging for this aspect of groups for as long.